core/domain/prod_validation_jobs_one_off.py

```diff
@@ -316,39 +380,56 @@ def _get_external_model_properties(cls):
         This should be implemented by subclasses.
 
         Returns:
-            tuple(str, list(tuple), dict). A tuple with first element as
-            external model name, second element as a tuple of
-            cls.external_instance_details and the third element
-            as a properties dict with key as property name in summary
-            model and value as property name in external model.
+            tuple(str, str, dict). A tuple with first element as
```

Comment:
Is it worth using ExternalModelReference here to simplify the return object?

Reply:
This is a different type - not related to ExternalModelReference and is only used for summary models where we want to compare properties in the summary model and the base model with which the summary model is associated.